src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -30,6 +30,7 @@ public override bool IsTriggerCharacter(SourceText text, int characterPosition,
                 ch == ' ' ||
                 ch == '[' ||
                 ch == '(' ||
+                ch == '~' ||
```

Comment:
"// Bring up on space or at the start of a word, or after a ( or [."

"Note: we don't want to bring this up after traditional enum operators"